src/main/webapp/js/map-loader.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** 
+ * Create the Google Map and center it around "Hogwarts" aka the Googleplex 
+ * @return the initialized map
+ */
+function createMap() {
+    //Create the Google Map.
+    const map = new google.maps.Map(document.getElementById("map"), {
+        center: {
+            lat: 37.422, 
```

Comment:
When coding I try my best to avoid shortening words like latitude, longitude, and message. Its not necessary, but it improves readability, makes the code less "codey"